libs/sdk-ui-kit/src/Table/useAsyncTableSearch.ts

```diff
@@ -0,0 +1,17 @@
+// (C) 2025 GoodData Corporation
+
+import { useEffect } from "react";
+import { useDebouncedState } from "../utils/debounce.js";
```

Comment:
`useDebouncedState` looks like something that can be useful even with other contexts (not only sdk-ui-kit)... what about moving it to sdk-ui ?